secant/Util/WalletStorage.swift

```diff
@@ -25,12 +34,105 @@ final class RecoveryPhraseStorage {
         case encoding
         case noDataFound
         case unknown(OSStatus)
-        case unsupportedVersion(Int)
-        case unsupportedLanguage(MnemonicLanguageType)
     }
     
-    // MARK: - Codable helpers
+    static func importWallet(
```

Comment:
Curious, why all the static functions?  Can we not use dependency injection?

Reply:
it doesn't, changed